src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -67,6 +67,10 @@ export default opts => {
     saleArtworksFilterLoader: gravityLoader("filter/sale_artworks"),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`, {}, { headers: true }),
     saleArtworkLoader: gravityUncachedLoader<any, { saleId: string, saleArtworkId: string }>(({ saleId, saleArtworkId }) => `sale/${saleId}/sale_artwork/${saleArtworkId}`, null),
+    saleArtworkCalculatedCostLoader: gravityLoader<any, { saleId: string, saleArtworkId: string, bidAmountMinor: number }>(
```

Comment:
@dblandin I recall @alloy [mentioning](https://artsy.slack.com/archives/CP9P4KR35/p1572287779181400?thread_ts=1572284881.180100&cid=CP9P4KR35) that this first type parameter could be used to define a type on the Gravity response.

Thoughts on this?

Reply:
Good call! Added. :+1: